src/mongo/dbtests/dbtests.cpp

```diff
@@ -156,6 +158,11 @@ int dbtestsMain(int argc, char** argv, char** envp) {
         ->setFollowerMode(repl::MemberState::RS_PRIMARY)
         .ignore();
 
+    repl::DropPendingCollectionReaper::set(
+                service,
+                std::make_unique<repl::DropPendingCollectionReaper>(new repl::StorageInterfaceMock())
+    );
+
```

Comment:
finally i moved storage mock to more suitable place (as far as i understand). @benety please review